app/views/eventos/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <h3>Editar evento</h3>
+  <%= simple_form_for @evento do |f| %>
+    <%= f.input :nome, required: true, label: "Nome" %>
+    <%= f.input :data_inicio, required: true, label: "Data início", as: :date, html5: true %>
+    <%= f.input :data_fim, required: true, label: "Data fim", as: :date, html5: true %>
+    <%= f.input :local, required: true, label: "Local" %>
+    <%= f.input :endereco, required: true, label: "Endereço" %>
+    <%= f.input :site, required: true, label: "Site" %>
+    <%= f.input :intro, label: "Introdução" %>
+    <%= f.input :modalidades, required: true, label: "Modalidades" %>
+    <%= f.input :cronograma, label: "Cronograma" %>
+    <%= f.input :inscricoes, required: true, label: "Inscrições" %>
+    <%= f.input :resultados, label: "Resultados" %>
+    <%= f.button :submit, "Salvar", class: "btn btn-success" %>
```

Comment:
Acho que não precisa você especificar todos os labels, só aqueles que são diferentes do automático. Por exemplo, :local vai virar Local, então não precisa colocar. Mas "introdução", "Inscrições" e "data fim" ele não vai fazer certo se deixar no automático.

Talvez seria melhor "Data de início" e "Data de término"